README.md

```diff
@@ -1 +1,205 @@
+
+To get a local copy up and running, follow these steps.
+- download the git repository 
+- run npm init -y to create a node_modules folder
+- add package.json to your root directory 
+- run npm install for install all dependencies.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- node 12.x or latest
+- npm 7.x or latest
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- cd my-folder
+- git clone git@github.com:Artufelii/microverse-hello-world
+
+### Install
+
+Install this project with:
+
+- cd microverse-hello-world
+- npm install
+
+### Usage
+
+To run the linters, execute the following command:
+
+- HTML: npx hint .
+- CSS: npx stylelint "**/*.{css,scss}"
+
+### Run tests
+
+To run tests, run the following command:
+
+- HTML: npx hint .
+- CSS: npx stylelint "**/*.{css,scss}"
+
+### Deployment
+
+You can deploy this project using:
+
+- You can deploy this web app directly to github pages or any other host that you want.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@Artufelii](https://github.com/Artufelii)
+- LinkedIn: [Arturo Alvarez](https://www.linkedin.com/in/arturo-alvarez-martinez-ab9b77151/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [ ] Nice one adding the MIT license 👏 . However, you need to have a file named `LICENSE` at the root of your project, with the content given on the README template, so the path here is not broken. 
-  [ ] Please do delete line `203`, as its meant to serve as a guide and not to be commented on your code 👍 . This keeps your code clean, and give a more professional feel